src/Components/Home.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import { Link } from "react-router-dom";
+
+function Home() {
+
+  const [articles, setArticles] = useState([])
+
+  useEffect(() => {
+    getArticles().then((articles) => {
+    console.log(articles)
+      setArticles(articles)
+    });
+  }, []);
+
+  return (
+    <div className="grid gap-[50px] grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
+        {articles.map((article) => {
+            return (
+                <Link to = {`/${article.article_id}`} >
```

Comment:
Could this be it's own component?

Reply:
I thought about it, it would definitely be more clean/semantic but articles are only going to exist on my home page so it won't be and dry(er)